Jac/amar kumar yadav 1044

jacalata opened the pull request:
not sure why the black/mypy jobs were failing for PR #1044 so pushing a new one in my name to check if that fixes it 🤷

amar-kumar-yadav commented:
Hi @jacalata , i see that you have removed "e" from permission_item.py line 51. Wouldn't it fail because when you do the API call for populating the permissions you add "s" after every item for eg. workbook's', flow's'. And for Lens it would add upto Lens's'  but the actual API call has Lens'es'. This is why i kept Lens = "lense" in permission_item.py line 51 rather than Lens = "Lens". Please let me know if my understanding is incorrect.

jacalata commented:
I missed that, you are correct. I've added sample code to validate the methods and updated that logic in a new pull request, since I accidentally merged this one to master anyway https://github.com/tableau/server-client-python/pull/1054

amar-kumar-yadav commented:
when can we expect the update in the master branch? @jacalata
